signature/policy_config_common.go

```diff
@@ -0,0 +1,8 @@
+//go:build !freebsd
```

Comment:
(Absolutely non-blocking: Naming this `pkg/sysregistriesv2/policy_paths_{common,freebsd}.go` would work just as well, and be a tiny bit more descriptive.)